scene/3d/path_3d.cpp

```diff
@@ -170,7 +172,18 @@ void Path3D::_bind_methods() {
 
 //////////////
 
-void PathFollow3D::_update_transform(bool p_update_xyz_rot) {
+void PathFollow3D::_update_transform(bool p_immediate) {
+	// Schedule update later.
+	if (!p_immediate) {
+		transform_dirty = true;
+		callable_mp(this, &PathFollow3D::_update_transform).call_deferred(true);
+		return;
+	}
```

Comment:
There should probably be 2 methods, one public with the `p_immediate` argument, and one private that is actually called if `p_immediate` is true or via this deferred call here.

Reply:
fixed